ROADMAP.md

```diff
@@ -0,0 +1,31 @@
+# ROADMAP
+
+# 1.6
+
+## Must-have features
+
+* Support for k8s 1.6
+* RBAC enabled by default
+* etcd version selection (probably etc3 by default)
```

Comment:
Typo: etcd3, maybe etcd3 for new clusters, probably don't auto-upgrade users though